### PR TITLE
Refactor-ProductCard

### PR DESCRIPTION
**Pull request by IvayloSlavchev**

### CODE REVIEW

**Review by JordanRad — APPROVED** [review]

**Review by Arabadzhiew — APPROVED** [review]

File: client/src/components/product-card/ProductCard.js [hunk 1]
```diff
@@ -1,21 +1,19 @@
-import './product-card.css';
+import React from 'react';
+import { Box, Typography, Button } from '@mui/material'
+import "./product-card.scss"
```
Comment (petya0111) [inline client/src/components/product-card/ProductCard.js:3]: add icon import 
```
import { IconButton } from '@mui/material';
import AddShoppingCartIcon from '@mui/icons-material/AddShoppingCart';
```

File: client/src/components/product-card/ProductCard.js [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
-                +
-            </button>
-        </div>
+            <Typography variant='h4' className="name">{props.name}</Typography>
+            <Box className='productInfo'>
+                <Typography>Sizes: (TODO?)</Typography>
+                <Typography>Small Description:  </Typography>
+                <Typography className="productPrice">{props.price}$</Typography>
+            </Box>
+            <Button className="button">+</Button>
```
Comment (petya0111) [inline client/src/components/product-card/ProductCard.js:15]: Please add a more accurate icon and stylize it to look more closer to the design
```
<IconButton className="button" sx={{ color: 'white', backgroundColor: 'rgba(69,37,242,1)'}}>
     <AddShoppingCartIcon>
     </AddShoppingCartIcon> 
</IconButton>
```

**Review by petya0111 — CHANGES_REQUESTED** [review]
The icon button needs to be improved. I tried one solution and there are several lines.

File: client/src/components/product-card/ProductCard.js [hunk 3]
```diff
@@ -1,21 +1,19 @@
...
+import { Box, Typography, Button } from '@mui/material'
+import "./product-card.scss"
 
 const ProductCard = (props) => {
-    
-
     return (
-        <div className="card">
+        <Box className='card'>
             <img src="http://www.foaminsulationni.com/images/placeholder/600x600.gif" alt="Image" className='productImage' />
```
Comment (petya0111) [inline client/src/components/product-card/ProductCard.js:8]: We should consider adding an image URL to the product model in the backend (also on the creation of the product endpoint) and map it in the frontend. To have a more completed design on the product page.

**Review by petya0111 — CHANGES_REQUESTED** [review]

**Review by petya0111 — APPROVED** [review]
Let's leave it as it is, I will do this in other task.

**Pull request merged**